python/pyspark/sql/pandas/conversion.py

```diff
@@ -34,7 +34,7 @@ class PandasConversionMixin(object):
     """
 
     @since(1.3)
-    def toPandas(self):
+    def toPandas(self, selfDestruct=False):
```

Comment:
Would this be better as an sql config? Since this class is a mixin, I'm not sure the user would see this option from the public api?

Reply:
I'll change it.